pkg/typesystem/typesystem.go

```diff
@@ -105,6 +105,7 @@ func Difference(base *openfgapb.Userset, sub *openfgapb.Userset) *openfgapb.User
 }
 
 type TypeSystem struct {
+	model           *openfgapb.AuthorizationModel
```

Comment:
todo: follow up and refactor the typsystem to work off of the model instead of storing `typeDefinitions`, `schemaVersion`, and `model`